src/Host/Broker/Impl/Sessions/SessionManager.cs

```diff
@@ -68,7 +69,8 @@ public class SessionManager {
                 password,
                 profilePath,
                 _loggingOptions.LogHostOutput ? _hostOutputLogger : null,
-                _loggingOptions.LogPackets ? _messageLogger : null);
+                _loggingOptions.LogPackets ? _messageLogger : null,
+                _loggingOptions.LogPackets ? LogVerbosity.Traffic : LogVerbosity.Minimal);
```

Comment:
If they both include traffic, shouldn't we set verbosity to `Traffic` when any of those properties are true?

---
In reply to: [81228732](https://github.com/Microsoft/RTVS/pull/2408#discussion_r81228732) [](ancestors = 81228732)